docs/components/dartpad/data_tables/normal/main.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: DataTableDemo(),
+    );
+  }
+}
+
+class DataTableDemo extends StatefulWidget {
+  @override
+  _DataTableDemoState createState() => _DataTableDemoState();
+}
+
+class _DataTableDemoState extends State<DataTableDemo> {
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Data Tables'),
+      ),
+      body: ListView(
+        padding: const EdgeInsets.all(16),
+        children: [
+          PaginatedDataTable(
+            header: Text('Header Text'),
+            rowsPerPage: 4,
+            columns: [
+              DataColumn(label: Text('Header A')),
+              DataColumn(label: Text('Header B')),
+              DataColumn(label: Text('Header C')),
+              DataColumn(label: Text('Header D')),
+            ],
+            source: _DataSource(context),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Row {
+  _Row(
+    this.headerA,
+    this.headerB,
+    this.headerC,
+    this.headerD,
+  );
+
+  final String headerA;
```

Comment:
Rename all these properties to `valueA`, `valueB`, etc. Since they represent the values in a row not the headers

Reply:
Done